frontends/PyCDE/src/pycde/value.py

```diff
@@ -52,7 +52,8 @@ def reg(self, clk, rst=None, name=None):
                            input=self.value,
                            clk=clk,
                            reset=rst,
-                           name=name)
+                           name=name,
+                           inner_sym=name)
```

Comment:
@teqdruid following the discussion in https://github.com/llvm/circt/pull/2303#pullrequestreview-826959487, I took the liberty of setting the inner_sym to name when it is provided here. I think this is what our users would expect, but let me know if you disagree. If they create the CompRegOp themselves, they still have the ability to have full control over name versus symbol.